Readme.md

```diff
@@ -4,13 +4,13 @@
 
 ## Overview
 
-[Hastlayer](https://hastlayer.com/) - be the hardware. Automatically transforming [.NET](https://www.microsoft.com/net/) assemblies into computer chips to improve performance and lower power consumption.
+[Hastlayer](https://hastlayer.com/) - Be the hardware. Hastlayer automatically transforms [.NET](https://www.microsoft.com/net/) assemblies into computer chips, improving performance and lowering power consumption.

```

Comment:
"be the hardware", all lowercase, is our slogan, so please keep that. The others are good suggestions.

Reply:
Thank you. I have committed the new changes.